.github/workflows/builder.yaml

```diff
@@ -0,0 +1,53 @@
+name: Builder
+
+on:
+  pull_request:
+
+jobs:
+  check_for_membership:
+    runs-on: ubuntu-latest
+    name: Check membership of given user
+    outputs:
+      check-result: ${{ steps.composite.outputs.check-result }}
+
+    steps:
+      - name: Action for membership check
+        id: composite
+        uses: hazelcast/hazelcast-tpm/membership@main
+        with:
+          organization-name: 'hazelcast'
+          member-name: ${{ github.event.pull_request.head.repo.owner.login }}
+          token: ${{ secrets.GITHUB_TOKEN }}
+  default:
+    needs: [check_for_membership]
+    if: needs.check_for_membership.outputs.check-result == 'true'
+    runs-on: ubuntu-latest
+    steps:
+      - name: Detect untrusted community PR
+        if: ${{ needs.check_for_membership.outputs.check-result == 'false' }}
+        run: |
+          gh pr comment ${{ github.event.number }} \
```

Comment:
I'm not actually sure we want a comment here, an annotation the job is probably enough.

(I know you copied it from the PR builder, which does that - but I deliberately used comments there for backwards compatibility reasons)

Not sold either way.

Reply:
Yes indeed, I copied as made sense to me 

> an annotation the job is probably enough

What do you mean? Like remove commenting and echo the error?

Reply:
>> an annotation the job is probably enough
> What do you mean? Like remove commenting and echo the error?

Just `echo`ing the error and then using something like this https://github.com/hazelcast/hazelcast-docker/pull/821